rudder-lang/tools/tester.sh

```diff
@@ -1,30 +1,27 @@
 #!/bin/bash
 
-set -xe # -x # to print every line as it occurs
+set -e # -x # to print every line as it occurs
 name=$1
-
-dir=/tmp/rudderc/tester/
-cfjson_tester=/opt/rudder/share/rudder-lang/tools/cfjson_tester
-technique_path=/var/rudder/configuration-repository/techniques/ncf_techniques/1.0/technique.cf
+dir=$PWD/tests/test_files/tester/$name
+cfjson_tester=$PWD/tools/cfjson_tester
 
 mkdir -p $dir/
 
 # Take original technique an make a json
-$cfjson_tester ncf-to-json $technique_path $dir/technique.json
+$cfjson_tester ncf-to-json $dir/technique.cf $dir/technique.json /tools/rudderc-dev.conf
```

Comment:
there should be no / at the beginning of tools/ it would imply that you put it in / which is misleading

Reply:
either use ./ or nothing